samza-core/src/main/java/org/apache/samza/util/ClassLoaderHelper.java

```diff
@@ -19,11 +19,30 @@
 
 package org.apache.samza.util;
 
+import org.apache.samza.config.ConfigException;
+
+import java.lang.reflect.Constructor;
+
 public class ClassLoaderHelper {
 
   public static <T> T fromClassName(String className) throws ClassNotFoundException, InstantiationException, IllegalAccessException {
     Class<T> clazz = (Class<T>) Class.forName(className);
     T instance = clazz.newInstance();
     return instance;
   }
+
+  public static <T> T fromClassName(String className, Class<T> classType) {
```

Comment:
I thought this was the new util class for Class Loading. I am aware that ApplicationRunner also has similar usage. I wasn't sure if I should change it as a part of this patch. Let me know if you think otherwise.